<html xmlns:py="http://genshi.edgewall.org/" xmlns:xi="http://www.w3.org/2001/XInclude" py:strip="">

<py:def function="announcement_id">
PMASA-2014-15
</py:def>

<py:def function="announcement_date">
2014-11-?
</py:def>

<py:def function="announcement_summary">
XSS vulnerability in error reporting functionality.
</py:def>

<py:def function="announcement_description">
With a crefted file name it is possible to trigger an XSS in the error reporting page.
</py:def>

<py:def function="announcement_severity">
We consider this vulnerability to be non critical.
</py:def>

<py:def function="announcement_mitigation">
This vulnerability can be triggered only by someone who is logged in to phpMyAdmin, as the usual token protection prevents non-logged-in users from accessing the required page.
</py:def>

<py:def function="announcement_affected">
Versions 4.1.x (prior to 4.1.14.7) and 4.2.x (prior to 4.2.12) are affected.
</py:def>


<py:def function="announcement_solution">
Upgrade to 4.1.14.7 or newer, or 4.2.12 or newer, or apply the patch listed below.
</py:def>

<py:def function="announcement_references">
Thanks to Johannes Dahse (<a href="https://twitter.com/FluxReiners">https://twitter.com/FluxReiners</a>) for reporting this vulnerability.
</py:def>

<py:def function="announcement_cve">CVE-2014-?</py:def>

<py:def function="announcement_cwe">661 79</py:def>

<py:def function="announcement_commits">
?
</py:def>

<py:def function="announcement_commits_4_1">
c641ad40c37bc562226c8a25cce77a273a07756b
</py:def>

<xi:include href="_page.tpl" />
</html>
